ui/component/headerCenterMenuButtons/view.jsx

```diff
@@ -0,0 +1,104 @@
+// @flow
+import 'scss/component/_header.scss';
+
+import { Link } from 'react-router-dom';
+import { Menu, MenuList, MenuButton, MenuItem, MenuLink } from '@reach/menu-button';
+import * as ICONS from 'constants/icons';
+import * as PAGES from 'constants/pages';
+import * as SETTINGS from 'constants/settings';
+import Icon from 'component/common/icon';
+import NotificationHeaderButton from 'component/headerNotificationButton';
+import React from 'react';
+import Tooltip from '@mui/material/Tooltip';
+
```

Comment:
- [ ] Replace all tooltips to use only 1 library -- avoid double library.

Don't want this to be a blocker, but if we are using MUI's tooltip, consider adding a task to replace all existing tooltips to MUI so that we don't bloat up the code.  Also add a task to re-do code splitting for MUI, as this change breaks the existing "suggestion.js" package.

Reply:
We already have `Tooltip` defined, and it's just a simple wrapper.  Maybe change there instead?  The next new person will be confused on which one to use.

This may or may not increase testing time though, since we are unused if the tooltip behavior between Reach and MUI is 1:1 or not.